src/main/java/uk/gov/digital/ho/pttg/application/HmrcRetryTemplateFactory.java

```diff
@@ -45,12 +47,14 @@ private CompositeRetryPolicy compositeRetryPolicy(SimpleRetryPolicy simpleRetryP
     }
 
     private SimpleRetryPolicy simpleRetryPolicy(int attempts) {
+        log.info("Retry policy has {} attempts", attempts);
```

Comment:
Needs an event ID

Reply:
I didn't view this as an "event", hence no event ID. It is really debug, but debug doesn't get picked up in prod ElasticSearch.